RELEASE-NOTES.md

```diff
@@ -10,14 +10,12 @@ The project release numbers follow [Semantic Versioning](http://semver.org/spec/
 ## [Unreleased] - Release date yyyy-mm-dd
 
 ## Added
-- Added variable BLT_CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES_EXCLUDE to allow removing
```

Comment:
The release notes had an outdated reference to `BLT_CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES_EXCLUDE` as well as a note about `BLT_CMAKE_IMPLICIT_LINK_DIRECTORIES_EXCLUDE`.  I combined these.